pymtl3/passes/rtlir/behavioral/test/BehavioralRTLIRPass_test.py

```diff
@@ -426,3 +426,29 @@ def update_ff_upblk():
         ] )
   }
   do_test( a )
+
+def test_fixed_size_slice( do_test ):
```

Comment:
Is this test included in TranslationImport test?

Reply:
Not yet. I'll add it to translation tests